### PR TITLE
Use --user-data-dir=some_tmp_location

### PR DESCRIPTION
**Pull request by atuchin-m**

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -82,9 +82,13 @@ const generateCRXFile = (binary, crxFile, privateKeyFile, publisherProofKey,
   if (!fs.existsSync(path.resolve(privateKeyFile))) {
     throw new Error(`Private key file '${privateKeyFile}' is missing, was it uploaded?`)
   }
+
+  const tmpDir = os.tmpdir();
```
Comment (antonok-edm) [inline lib/util.js:86]: note lint failures here

**Review by antonok-edm — APPROVED** [review]
I don't think any part of the CRX packaging process requires a populated profile directory, but let's verify it in staging too.

**Comment by atuchin-m** [comment]
> I don't think any part of the CRX packaging process requires a populated profile directory, but let's verify it in staging too.
The test job: https://ci.brave.com/job/brave-core-ext-ad-block-update-publish-dev/332/

File: lib/util.js (hunk 1 above)
Comment (atuchin-m) [reply lib/util.js]: Fixed

**Comment by antonok-edm** [comment]
Looks good from my testing. @atuchin-m shall we merge?

**Pull request merged**